Sources/StreamChat/Repositories/SyncRepository.swift

```diff
@@ -134,6 +137,12 @@ class SyncRepository {
                 )
             }
         operations.append(contentsOf: refetchChannelListQueryOperations)
```

Comment:
would we still need this line?

Reply:
I did not touch the controller related implementation at the moment. If we go for notification center based approach, the controller related implementation can be cleaned up as well and we would end up deleting this part. That should be a separate small PR then.

Reply:
I think that would be a duplicated logic, right? Good to remove it, so that we are sure that things work well with the new approach.

Reply:
I will create a separate follow-up PR for addressing it